grakn-test/src/test/java/ai/grakn/generator/TaskStates.java

```diff
@@ -79,7 +80,12 @@ public TaskState generate(SourceOfRandomness random, GenerationStatus status) {
         Json configuration = Json.object();
         TaskState taskState = TaskState.of(taskClass, creator, TaskSchedule.now(), configuration, taskId);
         configuration.set("id", taskState.getId().getValue());
-        return taskState.status(taskStatus);
+        if(taskStatus == TaskStatus.RUNNING){
+            taskState.setRunning(EngineID.me());
+        } else {
+            taskState.status(taskStatus);
```

Comment:
When a task changes status from no longer running does the EngineID on the task get wiped? Should it or shouldn't it/